lib/vsc/mympirun/mpi/openmpi.py

```diff
@@ -67,7 +95,7 @@ def pinning_override(self):
         self.log.debug("pinning_override: type %s ", override_type)
 
         ranktxt = ""
-        sockets_per_node = 2
+        sockets_per_node = self.determine_sockets_per_node()
```

Comment:
instead of sockets per node, shouldn't you be using the sockets available in the cpuset for the process?